src/app/(proper_react)/(redesign)/(public)/LandingView.stories.tsx

```diff
@@ -17,34 +15,18 @@ const meta: Meta<typeof View> = {
       props.experimentData ?? defaultExperimentData["Features"];
     const enabledFeatureFlags = props.enabledFeatureFlags ?? [];
     return (
-      <AccountsMetricsFlowProvider
```

Comment:
Traced all of this logic back to https://github.com/mozilla/blurts-server/pull/4822 which is Plus related.